src/internal/modules/charge-information/lib/charge-categories/constants.js

```diff
@@ -0,0 +1,111 @@
+'use-strict';
+
+const CHARGE_CATEGORY_STEPS = {
+  description: 'description',
+  source: 'source',
+  loss: 'loss',
+  volume: 'volume',
+  waterAvailability: 'water-availability',
+  waterModel: 'water-model',
+  isAdditionalCharges: 'additional-charges',
+  isAdjustments: 'adjustments'
+};
+
+const getStepKeyByValue = value => Object.keys(CHARGE_CATEGORY_STEPS).find(key => CHARGE_CATEGORY_STEPS[key] === value);
+
+const CHARGE_CATEGORY_FIRST_STEP = CHARGE_CATEGORY_STEPS.description;
+
+const SOURCES = {
+  tidal: 'tidal',
+  nonTidal: 'non-tidal'
+};
+
+const WATER_MODEL = {
+  noModel: 'no model',
+  tier1: 'tier 1',
+  tier2: 'tier 2'
+};
+
+const LOSS_CATEGORIES = {
+  high: 'high',
+  medium: 'medium',
+  low: 'low'
+};
+
+const WATER_AVAILABILITY = {
```

Comment:
I mentioned in the service pull request that I'm not 100% sure this value needs to be a string as it could simply be represented by true/false.

Not something I feel strongly about - just thought I'd mention it here too in case it's of interest.

Reply:
changed to boolean and renamed property to isRestrictedSource to match the water service